python/test/test_api_models/test/config.pbtxt

```diff
@@ -0,0 +1,28 @@
+name: "test"
```

Comment:
Removing use of `name` field in config will simplify re-use/renaming of this model/config for future tests. When specified, it must match the folder name. When unspecified, it just uses the folder name. So I find it pretty redundant and cumbersome for testing.